libcontainer/cgroups/systemd/v2.go

```diff
@@ -97,6 +99,9 @@ func (m *unifiedManager) Apply(pid int) error {
 		return cgroups.WriteCgroupProc(m.path, pid)
 	}
 
+	if m.rootless {
```

Comment:
nit: it would be better for readability to have it like
```go
slice := "system.slice"
if m.rootless {
  slice = "user.slice"
}
```

Reply:
updated